internal/pwru/config.go

```diff
@@ -79,6 +79,7 @@ func GetConfig(flags *Flags) FilterCfg {
 		cfg.FilterProto = syscall.IPPROTO_ICMP
 	case "icmp6":
 		cfg.FilterProto = syscall.IPPROTO_ICMPV6
+		cfg.FilterIPv6 = 1
```

Comment:
Should we check cfg.FilterIPv6 in the `if flags.FilterDstIP != ""` branch below and bail out in case it is set and the dst IP is an IPv4?